Explorer/Assets/DCL/UserInAppInitializationFlow/StartupOperations/CheckOnboardingStartupOperation.cs

```diff
@@ -46,10 +46,33 @@ public CheckOnboardingStartupOperation(
             this.realmNavigator = realmNavigator;
         }
 
+        public async UniTask MarkOnboardingAsDoneAsync(World world, Entity playerEntity, CancellationToken ct)
+        {
+            if (!isProfilePendingToBeUpdated || ownProfile == null || ownProfile.TutorialStep > 0)
+                return;
+
+            try
+            {
+                // Update profile data
```

Comment:
Please consider this [wiki](https://github.com/decentraland/unity-explorer/wiki/How-To#detached-flows) page since this is the direction proposed for async flows